src/@next/Icon/icons/icons.ts

```diff
@@ -59,6 +59,8 @@ export const iconNames = [
   'ri-facebook-circle-line',
   'ri-file-fill',
   'ri-file-line',
+  'ri-file-list-3',
```

Comment:
```suggestion
  'ri-file-list3-fill',
```

can we use this naming instead to follow convention. we'll try to improve documentation soon to help to guide with the naming format 🙇